src/hdf5plugin/__init__.py

```diff
@@ -390,9 +390,25 @@ class Zstd(_FilterRefClass):
             data=numpy.arange(100),
             **hdf5plugin.Zstd())
         f.close()
+
+    :param int clevel: Compression level from 1 (lowest compression) to 22 (maximum compression).
+        Ultra compression extends from 20 through 22. Default: 3.
+
+    .. code-block:: python
+
+        f = h5py.File('test.h5', 'w')
+        f.create_dataset(
+            'zstd',
+            data=numpy.arange(100),
+            **hdf5plugin.Zstd(clevel=22))
+        f.close()
     """
     filter_id = ZSTD_ID
 
+    def __init__(self, clevel=None):
+        if clevel is not None:
```

Comment:
An `assert` on `clevel` would be good: For the other filters we have `assert` on the input arguments when possible. We do so since error messages from hdf5 can be hard to understand for end-users.
Example for blosc: https://github.com/silx-kit/hdf5plugin/blob/main/src/hdf5plugin/__init__.py#L201

```suggestion
        if clevel is not None:
            assert 1 <= clevel <= 22
```